### PR TITLE
Player Buffs base

### PR DESCRIPTION
**Pull request by KyriakosNU**

### CODE REVIEW

**Review by Nijelous — APPROVED** [review]
This one is completely fine, please make sure the others also meet the coding standards.

**Review by ollieiscool — APPROVED** [review]
Seems succinct

File: CSC8503/InventoryBuffSystem/PlayerBuffs.cpp [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+			{
+				mOnBuffTickFunctionMap[entry->first](playerNo, dt);
+			}
+			else
+			{
+				RemoveBuffFromPlayer(entry->first, playerNo);
+			}
+		}
+	}
+}
```
Comment (fm19121) [inline CSC8503/InventoryBuffSystem/PlayerBuffs.cpp:63]: curly bracket on same line, like:
void PlayerBuffs::Update(float dt){
     .........
    for(){
        .......
  }
  if(){
     ......
  }
}

Comment (fm19121) [inline CSC8503/InventoryBuffSystem/PlayerBuffs.cpp:63]: curly bracket on same line, like:
void PlayerBuffs::Update(float dt){
      .........
    for(){
        .......
    }
    if(){
       ......
    }
}

**Pull request merged**